.github/workflows/dependencies.yml

```diff
@@ -12,17 +12,18 @@ jobs:
     steps:
       - name: Gather credentials
         id: credentials
-        uses: actions/create-github-app-token@v2.0.6
+        uses: actions/create-github-app-token@df432ceedc7162793a195dd1713ff69aefc7379e # v2.0.6
```

Comment:
note: Whoa, this is intense! I understand from a security point-of-view (pinning to a dash instead of a tag). Thanks for adding the comment!

question: How will dependabot handle this going forward? Will it try to use a tag?

Reply:
@mwbrooks I had the exact same questions! It's nice that we can keep this comment inline for somewhat reasonable understandings - I cannot remember hashes so well... 🔍 

> How will dependabot handle this going forward? Will it try to use a tag?

AFAICT tags from official **releases** will be used in these updates and the comment that follows will match. This should match the current updating events, but instead with commit details!

An [example](https://github.com/slackapi/slack-github-action/pull/443/files) update has shown this to be alright and this [blog](https://github.blog/changelog/2022-10-31-dependabot-now-updates-comments-in-github-actions-workflows-referencing-action-versions/) post shared a few more detail 🎃